clients/firestore/lib/google_api/firestore/v1beta1/connection.ex

```diff
@@ -0,0 +1,83 @@
+# Copyright 2017 Google Inc.
```

Comment:
The year should be 2018. I realize it's hard-coded in the code generator and I'll need to come up with a global fix (it should be when the file is first created which will be hard to determine when we re-generate clients).

If you don't get to it, I'll fix it before a release.

Reply:
Fixing!